search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/aggregation/result/UsageMetricsTUAggregationResultTransformer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients.transformers.aggregation.result;
+
+import io.camunda.search.aggregation.UsageMetricsAggregation;
+import io.camunda.search.aggregation.result.UsageMetricsTUAggregationResult;
+import io.camunda.search.clients.core.AggregationResult;
+import io.camunda.search.entities.UsageMetricTUStatisticsEntity;
+import io.camunda.search.entities.UsageMetricTUStatisticsEntity.UsageMetricTUStatisticsEntityTenant;
+import io.camunda.search.entities.UsageMetricTUStatisticsEntity.UsageMetricTUStatisticsEntityTenant.Builder;
+import io.camunda.webapps.schema.entities.metrics.UsageMetricsEventType;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class UsageMetricsTUAggregationResultTransformer
+    implements AggregationResultTransformer<UsageMetricsTUAggregationResult> {
+
+  private long extractMetricCount(
+      final AggregationResult eventTypes, final UsageMetricsEventType metricName) {
```

Comment:
🔧 I guess here the metricName parameter is not strictly needed.

```suggestion
  private long extractTUMetricCount(final AggregationResult eventTypes) {
```